v3/cypress/support/elements/component-elements.ts

```diff
@@ -12,7 +12,7 @@ export const ComponentElements = {
     graphDisplayValuesButton: "Change what is shown along with the points",
     graphDisplayConfigButton: "Configure the display differently",
     graphDisplayStylesButton: "Change the appearance of the display",
-    graphCameraButton: "Save the image as a PNG file",
+    graphCameraButton: "Save the image",
```

Comment:
This change was required to get the `checks all graph tooltips` test in v3/cypress/e2e/graph.spec.ts to pass. The change to the related translation string value was made in [this PR](https://github.com/concord-consortium/codap/pull/995). I'm not sure why the test didn't fail there.